specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-11-01/netapp.json

```diff
@@ -0,0 +1,4890 @@
+            "schema": {
+              "$ref": "#/definitions/netAppAccountList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Accounts_List": {
+            "$ref": "examples/Accounts_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NetApp/netAppAccounts/{accountName}": {
+      "get": {
+        "summary": "Describe a NetApp Account",
+        "description": "Get the NetApp account",
+        "tags": [
+          "NetApp Accounts"
+        ],
+        "operationId": "Accounts_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/netAppAccount"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Accounts_Get": {
+            "$ref": "examples/Accounts_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "NetApp Accounts"
+        ],
+        "operationId": "Accounts_CreateOrUpdate",
+        "summary": "Create or update a NetApp account",
+        "description": "Create or update the specified NetApp account within the resource group",
+        "parameters": [
+          {
+            "name": "body",
+            "description": "NetApp Account object supplied in the body of the operation.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/netAppAccount"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroup"
+          },
+          {
+            "$ref": "#/parameters/AccountName"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - account updated",
+            "schema": {
+              "$ref": "#/definitions/netAppAccount"
+            }
+          },
+          "201": {
+            "description": "Account created",
+            "schema": {
+              "$ref": "#/definitions/netAppAccount"
+            }
+          },
+          "202": {
+            "description": "Accepted -- Create, update or delete request accepted; operation will complete asynchronously"
```

Comment:
Why you have 'delete' as part of the description? Also, when is your service returning either a 201 or 202?
According to our RPC, for async PUT calls the recommendation is to return 201 or 200, for async Delete calls return 202.

Reply:
While this is a good catch a small note that this is unchanged in this version. I am not sure where this came from has been in for a long time now probably form the beginning. The service does indeed return 201 here, I would be happy to fix this.